### ISSUE TITLE
Logging configuration extends beyond the scope of CLISOPS

### ISSUE THREAD

**Issue by Zeitsperre** [roocs/clisops#144]
* clisops version: 0.6.1
* Python version: Python3.7
* Operating System: Linux

### Description

I think the logging configuration for CLISOPS might be a bit too aggressive. I'm noticing the logging call layout in projects that import CLISOPS but aren't specifically calling its functions. It could be that this needs behaviour to be explicitly restricted to the scope of this library to prevent it from showing up where it shouldn't.

### What I Did

See below. Notice the warnings formatted using the layout specified in 
`clisops/clisops/etc/logging.conf`:
```
$ pytest tests
...
tests/test_geo_utilities.py::TestFileInfoFuncs::test_single_file_check 2021-03-10 14:07:18,341 - RavenPy - ERROR - No files found. Exiting.
2021-03-10 14:07:18,341 - RavenPy - ERROR - Multi-file handling for file is not supported. Exiting.
PASSED
tests/test_geo_utilities.py::TestFileInfoFuncs::test_boundary_check 2021-03-10 14:07:18,355 - RavenPy - WARNING - Vector /home/USER/gh/raven-testdata/nasa/Mars_MGS_MOLA_DEM_georeferenced_region_compressed.tiff contains geometries in high latitudes. Verify choice of projected CRS is appropriate for analysis.
2021-03-10 14:07:18,361 - RavenPy - WARNING - Vector /home/USER/gh/raven-testdata/polygons/mars.geojson contains geometries in high latitudes. Verify choice of projected CRS is appropriate for analysis.
2021-03-10 14:07:18,368 - RavenPy - WARNING - Vector /home/USER/gh/raven-testdata/nasa/Mars_MGS_MOLA_DEM_georeferenced_region_compressed.tiff contains geometries in high latitudes. Verify choice of projected CRS is appropriate for analysis.
2021-03-10 14:07:18,370 - RavenPy - ERROR - Unable to read boundaries from unreal.zip
PASSED
...
```

**Comment by Zeitsperre** [comment]
For more context, this can also be seen in otherwise squelched warnings made visible when importing `clisops` before `xclim`:
```
from xclim import atmos
2021-03-11 15:58:14,790 - pint.util - WARNING - Redefining 'degree_Celsius' (<class 'pint.definitions.UnitDefinition'>)
2021-03-11 15:58:14,791 - pint.util - WARNING - Redefining '°C' (<class 'pint.definitions.UnitDefinition'>)
2021-03-11 15:58:14,792 - pint.util - WARNING - Redefining 'celsius' (<class 'pint.definitions.UnitDefinition'>)
2021-03-11 15:58:14,792 - pint.util - WARNING - Redefining 'degC' (<class 'pint.definitions.UnitDefinition'>)
2021-03-11 15:58:14,793 - pint.util - WARNING - Redefining 'degreeC' (<class 'pint.definitions.UnitDefinition'>)
2021-03-11 15:58:14,793 - pint.util - WARNING - Redefining 'C' (<class 'pint.definitions.UnitDefinition'>)
2021-03-11 15:58:14,794 - pint.util - WARNING - Redefining 'deg_C' (<class 'pint.definitions.UnitDefinition'>)
2021-03-11 15:58:14,795 - pint.util - WARNING - Redefining '[speed]' (<class 'pint.definitions.DimensionDefinition'>)
```

**Comment by Zeitsperre** [comment]
Closed with #216
